src/trie/rune_util.h

```diff
@@ -63,6 +63,13 @@ static inline void runeBufFree(runeBuf *buf) {
   }
 }
 
+static inline void printfRune(const rune *rune, size_t len) {
```

Comment:
Do we use it somewhere? if not lets delete, if we do lets use RM_Log.

Reply:
it is nice to have for debug since it is hard to read `rune` itself.

Reply:
Do you think we might need it to debug production issues? If so lets make a debug command out of it?

Reply:
This is internal. The print out to the cli will be identical to the original. It is more useful when we scan it and try to understand what is the current state.